src/app/_services/form-constants.service.ts

```diff
@@ -257,11 +257,11 @@ export class FormConstantsService {
     { code: 'ZMB', name: 'Zambia' },
     { code: 'ZWE', name: 'Zimbabwe' }
   ];
-  private categories = ['Non-Profit'];
+  private categories = [{name: 'Non-Profit', value: 'N'}];
 
   constructor() { }
 
-  getCategories(): string[] {
+  getCategories(): any[] {
```

Comment:
private categories = [{name: 'Non-Profit', value: 'N'}];

Categories is actually constant array with key value pairs in it so I don't know how to define type of categories.